src/main/java/seedu/duke/Storage.java

```diff
@@ -46,66 +49,86 @@ public void checkDirectory() {
     /**
      * Checks if the property file exist.
      */
-    public void checkPropertyFile() {
+    public void checkPropertyFile(PropertyList propertyList) {
         File propertyFile = new File(PROPERTY_PATH);
         boolean hasPropertyFile = propertyFile.exists();
 
         if (hasPropertyFile) {
             // Load property into the array list
-            loadProperty(propertyFile);
+            loadProperty(propertyList, propertyFile);
         }
     }
 
     /**
      * Checks if the client file exist.
      */
-    public void checkClientFile() {
+    public void checkClientFile(ClientList clientList) {
         File clientFile = new File(CLIENT_PATH);
         boolean hasClientFile = clientFile.exists();
 
-        if (!hasClientFile) {
+        if (hasClientFile) {
             // Load client into the array list
-            loadClient(clientFile);
+            loadClient(clientList, clientFile);
```

Comment:
Based on SLAP, I think it would be more appropriate if loadClient() were not called by checkClientFile(). You could create another method called loadFile() which calls loadClient()